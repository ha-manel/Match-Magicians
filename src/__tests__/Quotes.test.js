import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';
import '@testing-library/jest-dom';

describe('Test of quotes page', () => {
  it('test if quotes page is rendered', () => {
    const quotePage = renderer
      .create(<Quote />)
      .toJSON();
    expect(quotePage).toMatchSnapshot();
  });
});
