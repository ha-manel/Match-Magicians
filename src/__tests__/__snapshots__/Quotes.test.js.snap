// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of quotes page test if quotes page is rendered 1`] = `
<div
  id="quote"
>
  <ul>
    <li>
      "Mathematics is the music of reason."
      <span>
        -James Joseph Sylvester
      </span>
    </li>
    <li>
      "‘Obvious’ is the most dangerous word in mathematics."
      <span>
        -Eric Temple Bell
      </span>
    </li>
    <li>
      "Mathematics allows for no hypocrisy and no vagueness."
      <span>
        -Stendhal
      </span>
    </li>
    <li>
      "It is impossible to be a mathematician without being a poet in soul."
      <span>
        -Sofia Kovalevskaya
      </span>
    </li>
    <li>
      "Go down deep enough into anything and you will find mathematics."
      <span>
        -Dean Schlicter
      </span>
    </li>
  </ul>
</div>
`;
